### PR TITLE
Particles on lava

### PR DESCRIPTION
**Pull request by AdSand**
I think we have all seen this code enough by now.

Add a new particle system in the LoadParticleSystems() function of GameplayState.
Add it to the list of particlesystems.
This will be passed to renderer, and it will do everything for you.

### CODE REVIEW

File: Client/States/GameplayState.cpp [hunk 1]
```diff
@@ -222,6 +226,26 @@ void GameplayState::InitTimerBar(){
...
+
+    
+
+
+    renderer->PassParticleSystems(particleSystems);
+}
+
+void GameplayState::UpdateParticleSystems(float dt)
+{
+    for (auto ps : particleSystems)
```
Comment (Odycious) [inline Client/States/GameplayState.cpp]: & for ps

File: Client/States/GameplayState.h [hunk 2]
```diff
@@ -249,6 +250,12 @@ namespace NCL {
             void UpdateTimerBox(Element& element, float dt);
             void UpdateTimerText(Element& element, float dt);
             void UpdateTimerNub(Element& element, float dt);
+
+            ParticleSystem* lavaParticles;
+            vector<ParticleSystem*> particleSystems;
```
Comment (Odycious) [inline Client/States/GameplayState.h:272]: std::vector<>

**Review by Odycious — APPROVED** [review]
Particle system looks good to me, just a couple sections of white to clear up and we're good!

**Comment by AdSand** [comment]
Changes made

**Pull request merged**